packages/core/src/CloseButton/__snapshots__/CloseButton.spec.js.snap

```diff
@@ -27,8 +27,6 @@ exports[`CloseButton renders without props 1`] = `
   border-radius: 2px;
   border-width: 0;
   border-style: solid;
-  font-size: 14px;
-  padding: 9.5px 18px;
```

Comment:
Is this snapshot update expected?

Reply:
I'm not sure why this got removed but these styles have zero impact on this component, the padding is overwritten below and the font-size doesn't affect the icon.